meinberlin/apps/livequestions/assets/CategorySelect.jsx

```diff
@@ -1,5 +1,6 @@
 import django from 'django'
 import React from 'react'
+import $ from 'jquery' // FIXME needed to run test but file should be refactored to not include jquery
```

Comment:
added this as attempted several other options but without success:
- import or require via setup-jest file
- unmock and import into test file
- import or require into test file
- setting up basic jquery mock
- update funtion to not use jquery
as we want to remove jquery anyway i felt this was best option for now

Reply:
ah i see, yes probably jQuery is too hard to mock. And refactoring to get rid of it just to write a test might be too much effort. Lets leave it for now.